src/test/java/mil/dds/anet/test/resources/RollupGraphComparatorTest.java

```diff
@@ -0,0 +1,71 @@
+package mil.dds.anet.test.resources;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import mil.dds.anet.beans.RollupGraph;
+import mil.dds.anet.resources.ReportResource.RollupGraphComparator;
+import mil.dds.anet.test.TestData;
+
+public class RollupGraphComparatorTest {
+
+	@Test
+	public void performTest() {
+		
+		List<RollupGraph> rollupGraphs = new ArrayList<>();
+		
+		RollupGraph rollupGraph1 = TestData.createRollupGraph();
+		rollupGraph1.getOrg().setLongName("a name");
```

Comment:
I would switch the order of "a name" and "b name" to test the fallback to the natural order